config/rbac/operator-role.yaml

```diff
@@ -80,6 +80,13 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
What about adding it in the `camel-k-operator-knative` role, as it's strictly needed for Knative, to keep the RBAC modularity consistent?

Reply:
I thought about it, but having the `apiGroup: ""` not in any of the knative objects, then maybe this role could be set in the camel-k-operator, but having it as you suggest would be fine too.

Reply:
I see what you mean. Then having a new role with an explicit name, so it's clear it's not part of the core / mandatory roles, would work even better IMO.

Reply:
For me it's fine having the permission defined in operator-knative.yaml, in the end it's related to the knative interoperability, then it's fine to be there.

Reply:
Sounds good to me too.